### DISCUSSION TITLE
File upload expire

### DISCUSSION THREAD

**Discussion by naingwin** [filamentphp/filament#6862] [Help]
![Screenshot from 2023-06-28 14-59-08](https://github.com/filamentphp/filament/assets/131633682/b1e3d9d1-1dd9-4ea6-b053-946bd7096114)

How can I remove expires parameter in RichEditor upload file? After uploaded to amazon s3, the returned file url include X-Amz-Expires=300 parameter so the picture expire in 5 minutes.

**Comment by danharrin** [comment]
It's because its a temporary URL. Fix it in the Livewire config `temporary_file_upload.max_upload_time`
